vignettes/xcms.Rmd

```diff
@@ -1285,6 +1284,134 @@ identification of e.g. features with significant different
 intensities/abundances it is suggested to use functionality provided in other R
 packages, such as Bioconductor's excellent `limma` package.
 
+## Alignment to an external reference dataset
+
+In certain experiments, aligning two different datasets is necessary. This
+can involve comparing runs of the same samples conducted across different
+laboratories or runs with MS2 recorded after the initial MS1 run. Across
+laboratories and over time, the same samples may result in variation in
+retention time, especially because the LC system can be quite unstable. In these
+cases, an alignment step using the `adjustRtime()` function with the
+`LamaParam` parameter can allow the user to perform this type of alignment.
+We will go through this step by step below. 
+
+Let's load an already analyzed dataset `ref` and our previous dataset before
+alignment, which will be `tst`. We will first restrict their retention time
+range to be the same for both dataset.
+
+```{r}
+ref <- loadXcmsData("xmse")
+tst <- loadXcmsData("faahko_sub2")
+```
+
+Now, we will try to align these two samples to the previous dataset. The first
+step is to extract landmark features (called `lamas`). For this, we will
+determine the features that are present in every sample of the `ref` dataset
+using the `PercentMissingFilter()` parameter in the `filterFeatures()` function.
+
+```{r}
+f <- sampleData(ref)$sample_type
```

Comment:
maybe this definition of `f` might be a bit misleading/confusing if not explained? If you say that we want to identify features that are present in **every** sample of `ref`, that it would maybe be better to use `f <- rep(1, length(ref))`? with your code you are selecting features that are present in all samples, except QC samples.

Reply:
Actually for this alignment technique purpose, should we encourage people to maybe base it on features present in all QCs samples ? 
´f <- rep(1, length(ref))` works perfectly for this example but no real experiment will be like that with QCs and one sample type only no ?

Reply:
indeed - but then I would maybe a) describe that, and b) use just the QCs and not the other samples (at present you use just the study samples and exclude the QCs).

Reply:
Yes okay sounds good i'll adjust that !